compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/NodeCloner.java

```diff
@@ -1882,7 +1882,11 @@ public void visit(BLangSimpleVarRef.BLangFieldVarRef fieldVarRef) {
 
     @Override
     public void visit(BLangSimpleVarRef.BLangPackageVarRef packageVarRef) {
-        // Ignore
+        BLangSimpleVarRef.BLangPackageVarRef clone = new BLangSimpleVarRef.BLangPackageVarRef(
+                (BVarSymbol) packageVarRef.varSymbol);
```

Comment:
I'm not sure we should be using the symbol from the source here. @hasithaa, thoughts please?

Reply:
We discussed this and decided to go ahead with this change for now since this is a desugared construct.